genesyscloud/architect_flow/resource_genesyscloud_architect_flow_utils.go

```diff
@@ -35,8 +41,112 @@ func GenerateFlowResource(resourceLabel, srcFile, fileContent string, forceUnloc
 	return flowResourceStr
 }
 
-// setFileContentHashToNil This operation is required after a flow update fails because we want Terraform to detect changes
-// in the file content hash and re-attempt an update, should the user re-run terraform apply without making changes to the file contents
+// architectFlowResolver downloads and processes an architect flow from Genesys Cloud.
+// It creates a subdirectory, downloads the flow file, and updates the resource configuration.
+//
+// Parameters:
+//   - flowId: The ID of the architect flow to resolve
+//   - exportDirectory: The base directory where the flow will be exported
+//   - subDirectory: The subdirectory name to create within the export directory
+//   - configMap: Configuration map containing resource settings
+//   - meta: Provider metadata containing client configuration
+//   - resource: Resource information for the architect flow
+//
+// Returns:
+//   - error: Returns an error if any operation fails, nil otherwise
+func architectFlowResolver(flowId, exportDirectory, subDirectory string, configMap map[string]any, meta any, resource resourceExporter.ResourceInfo) (err error) {
+	defer func() {
+		if err != nil {
+			err = fmt.Errorf("caught in architectFlowResolver: %w", err)
+		}
+	}()
+
+	var (
```

Comment:
Why use a var block here.  Usually a var block is only used when you are not initialziing a value.  In each of these cases an := would be sufficient.

I think you have been hanging out in Typescript land to long Charlie :)